gbtis/CanvasWindow.xaml.cs

```diff
@@ -85,6 +85,11 @@ public partial class CanvasWindow : Window {
             helpButton.Clicked += (s, e) => Help?.Invoke();
         }
 
+        //im so sorry
+        public void clearScreen() {
+            canvas.Strokes.Clear();

```

Comment:
so i can clear the screen without making a new one everytime i need to access a new one (if you want to remake it, go ahead but i need a way to do it)